app/views/user/show.html.erb

```diff
@@ -1 +1,37 @@
-<h1>User Details</h1>
\ No newline at end of file
+  
+<div class="card mb-3">
+  <div class="row g-0">
+    <div class="col-md-4">
+      <%= link_to user_path(@user) do %>
+        <img src="<%= @user.Photo %>" class="img-fluid img-thumbnail" alt="User Profile">
+      <% end %>
+    </div>
+    <div class="col-md-8">
+      <div class="card-body">
+        <%= link_to user_path(@user.id) do %>
+          <h5 class="card-title"><%= @user.Name %></h5>
+        <% end %>
+        <p class="card-text"><small class="text-muted">Number of posts: <%= @user.posts.count %></small></p>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="card">
+  <div class="card-body">
+    <h6 class="card-title">Bio</h6>
+    <p class="card-text"><%= @user.Bio %></p>
+  </div>
+</div>
+<% @user.posts.each.with_index(1) do |post, index| %>
+  <div class="card text-center">
+    <div class="card-header">
+      <%= @post.Title %>
+      <p class="card-text"><small class="text-muted">Comments: <%= @post.CommentsCounter %></small></p>
+      <p class="card-text"><small class="text-muted">Likes: <%= @post.LikesCounter %></small></p>
+    </div>
+    <div class="card-body">
+      <p class="card-text"><%= @post.Text %></p>
+      <%= link_to "See Post", user_post_path(post.author, post), class: "btn btn-primary" %>
+    </div>
+  </div>
+<% end %>
```

Comment:
- [ ] You are doing great, congrats! However, I'm getting an error here:

![Screen Shot 2022-09-20 at 2 37 38 PM](https://user-images.githubusercontent.com/83717285/191350417-22a97ecb-740c-46ea-b3b5-546ec6350ee6.png)

I would suggest checking out what's the difference between `@post.Title` and `post.Title` inside an `.each` loop 😉